spec/contentful/bootstrap/commands/create_space_spec.rb

```diff
@@ -18,7 +18,7 @@
       end
 
       it 'does not create template when template_name is nil' do
-        create_space_command = subject.class.new(token, 'foo', nil, 'baz', false)
+        create_space_command = described_class.new(token, 'foo', nil, 'baz', false)
```

Comment:
It's just to respect rspec standards, as i was adding stuff, i thought of it as a nice extra touch